tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -39,9 +39,29 @@ export async function prepareProjectForE2e(name) {
 
   console.log(`Project ${name} created... Installing packages.`);
   await installWorkspacePackages();
+  await installPackage('puppeteer');
```

Comment:
pin this to an exact version;

This can be done in `projects-root/node_modules` so that all "test-projects" can access it via node_modules resolution.